src/Files.App/Filesystem/Security/AccessControlEntry.cs

```diff
@@ -13,56 +13,144 @@
 namespace Files.App.Filesystem.Security
 {
 	/// <summary>
-	/// Represents an ACE.
+	/// Represents an access control entry (ACE).
 	/// </summary>
 	public class AccessControlEntry : ObservableObject
 	{
-		public bool IsFolder { get; set; }
-
+		/// <summary>
+		/// Whether the path indicates folder or not
+		/// </summary>
+		public bool IsFolder { get; private set; }
+
+		/// <summary>
+		/// The owner in the security descriptor (SD).
+		/// NULL if the security descriptor has no owner SID.
+		/// </summary>
 		public Principal Principal { get; set; }
 
-		private AccessControlType _AccessControlType;
-		public AccessControlType AccessControlType
+		/// <summary>
+		/// Whether the ACE is inherited or not
+		/// </summary>
+		public bool IsInherited { get; private set; }
+
+		/// <summary>
+		/// Whether the ACE is editable or not
+		/// </summary>
+		public bool IsEditable
+			=> IsSelected && !IsInherited && false;
+
+		/// <summary>
+		/// AccessControlTypeHumanized
+		/// </summary>
+		public string AccessControlTypeHumanized
+			=> AccessControlType switch
+			{
+				AccessControlEntryType.Allow => "Allow",
+				_ => "Deny" // AccessControlType.Deny
```

Comment:
If we comment the value on the side, better put it in the switch directly?

Reply:
Don't think so. If we put AccessControlType.Deny directly, VS suggests you should add `_` switch case.